### DISCUSSION TITLE
DFP-34X-2C2: lost IP access after erase mtd3 partition

### DISCUSSION THREAD

**Discussion by hcteixeira** [Anime4000/RTL960x#103] [Q&A] [answered]
Hi, I've lost IP access to DFP-34X-2C2 after a reset erasing mtd3 partition. Is this an expected behavior?

**Comment by Anime4000** [comment, marked as answer]
erase mtd3 (factory reset) will use factory settings, you should use default IP Address 192.168.1.1/24 to connect back to ODI Stick.

**Comment by hcteixeira** [comment]
My mistake... I tried several times without success, but now IP 192.168.1.1 is responding.

I actually reseted the device due to strange behavior: after a successful connection, every 360 seconds I've a 15 seconds "blackout", with no connection response. Have you ever seen anything like this?

      0 8.8.8.8                                    56 121 25ms 
      1 8.8.8.8                                    56 121 27ms 
      2 8.8.8.8                                    56 121 25ms 
    ...
    129 8.8.8.8                                    56 121 25ms 
    130 8.8.8.8                                    56 121 25ms 
    131 8.8.8.8                                                 timeout                                  
    132 8.8.8.8                                                 timeout                                  
    133 8.8.8.8                                                 timeout                                  
    134 8.8.8.8                                                 timeout                                  
    135 8.8.8.8                                                 timeout                                  
    136 8.8.8.8                                                 timeout                                  
    137 8.8.8.8                                                 timeout                                  
    138 8.8.8.8                                                 timeout                                  
    139 8.8.8.8                                                 timeout                                  
    140 8.8.8.8                                                 timeout                                  
    141 8.8.8.8                                                 timeout                                  
    142 8.8.8.8                                                 timeout                                  
    143 8.8.8.8                                                 timeout                                  
    144 8.8.8.8                                                 timeout                                  
    145 8.8.8.8                                    56 121 25ms 
    146 8.8.8.8                                    56 121 25ms 
    147 8.8.8.8                                    56 121 25ms 
    ...
    488 8.8.8.8                                    56 121 25ms 
    489 8.8.8.8                                    56 121 25ms 
    490 8.8.8.8                                                 timeout                                  
    491 8.8.8.8                                                 timeout                                  
    492 8.8.8.8                                                 timeout                                  
    493 8.8.8.8                                                 timeout                                  
    494 8.8.8.8                                                 timeout                                  
    495 8.8.8.8                                                 timeout                                  
    496 8.8.8.8                                                 timeout                                  
    497 8.8.8.8                                                 timeout                                  
    498 8.8.8.8                                                 timeout                                  
    499 8.8.8.8                                                 timeout                                  
    500 8.8.8.8                                                 timeout                                  
    501 8.8.8.8                                                 timeout                                  
    502 8.8.8.8                                                 timeout                                  
    503 8.8.8.8                                                 timeout                                  
    504 8.8.8.8                                    56 121 25ms 
    505 8.8.8.8                                    56 121 25ms 
    506 8.8.8.8                                    56 121 27ms 
    ...

**Reply by hcteixeira** [reply]
![image](https://user-images.githubusercontent.com/13389061/205517781-3c686ae7-8a5c-4e7d-9874-a629112e6078.png)

**Reply by Anime4000** [reply]
problably ODI Stick still undergoes SFP SGMII Auto Negotiation, maybe you tried which 1Gb LAN_SDS mode that your mikrotik like:
![image](https://user-images.githubusercontent.com/1908715/205556649-a77057d5-5a22-41df-92c0-f5b7350e4c32.png)

**Reply by hcteixeira** [reply]
I tried with modes 0, 1, 2 and 3, without success. Always the same behavior... It seems that after a longer connection the packet loss increases. Maybe it's a faulty unit anyway?

**Reply by Anime4000** [reply]
seem like it, if you have SFP Ethernet Media Converter, try that if same issue

**Reply by hcteixeira** [reply]
I don't have a SFP Ethernet Media Converter, but I'll try other tests. Thanks!

**Reply by Anime4000** [reply]
it would be good if you have Media Converter, issue like this easy to troubleshoot

**Reply by hcteixeira** [reply]
@Anime4000 I've tried with another unit and got the same behavior, so I've bought a Media Converter and same result again (from Mikrotik hEX S eth port). My connection with ISP is a PPPoE over the link, any advice?
